basiclti/tsugi-util/src/java/org/tsugi/time/InstantUtil.java

```diff
@@ -0,0 +1,65 @@
+package org.tsugi.time;
+
+import java.util.Date;
+import java.time.Instant;
+import java.text.SimpleDateFormat;
+import java.time.format.DateTimeParseException;
+
+import org.apache.commons.httpclient.util.DateUtil;
+import org.apache.commons.httpclient.util.DateParseException;
+
+public class InstantUtil {
```

Comment:
Unit test would be ideal here

Reply:
There is a unit test

./tsugi-util/src/test/org/tsugi/time/InstantUtilTest.java